.serena/memories/nuxt4-hono-tanstack-query-project-overview.md

```
@@ -1,67 +1,130 @@
-- **ESLint** - 静的解析
-- **Prettier** - フォーマッター
-- **@hey-api/openapi-ts** - 型定義自動生成
+- **Biome** (2.2.3) - リンター・フォーマッター
+- **ESLint** (9.35.0) - 静的解析
+- **Prettier** (3.6.2) - フォーマッター
+- **@hey-api/openapi-ts** (0.82.4) - 型定義自動生成
 
-## プロジェクト構造
+## 実際のプロジェクト構造
 
 ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)